apps/design-system/src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import type { Meta } from "@storybook/react"
+import {
+  TextArea,
+  Field,
+  Form,
+  Group,
+  Fieldset,
+  IconButton,
+  TextInput,
+  Label,
+  Dropdown,
+  ChipInput,
+} from "@asyncapi/studio-ui"
+import { AddIcon, TrashIcon } from "@asyncapi/studio-ui/icons"
+
+const meta: Meta<typeof Form> = {
+  component: Form,
+  parameters: {
+    layout: "fullscreen"
+  },
+}
+
+export default meta
+
+const singleSelectOptions = [
+  { label: "HTTP", value: "http" },
+  { label: "Kafka", value: "kafka" },
+  { label: "Websocket", value: "websocket" },
+  { label: "AMQP", value: "amqp" },
+  { label: "MQTT", value: "mqtt" },
+  { label: "Google PubSub", value: "googlepubsub" },
+  { label: "IBM MQ", value: "ibmmq" },
+  { label: "NATS", value: "nats" },
+  { label: "Pulsar", value: "pulsar" },
+]
+export const Default = () => (
+  <Form
+    title="User Registration"
```

Comment:
```suggestion
    title="User Registration"  aria-label="User Registration Form"
```
Adding aria-label for accessibility.

Reply:
He's already giving it an aria-label here: https://github.com/asyncapi/studio/pull/773/files/002b3879480e31fcec954fe12b92973dda2c125e#diff-5a1a91d42d491ef6f46dea784b0e1792139d6e5bd79158f152d5c67cdc836b85R10.